doc/Command-Reference.md

```diff
@@ -1730,6 +1734,76 @@ This command is used to remove particular IPv4 or IPv6 BGP neighbor configuratio
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#bgp)
 
+## Container Auto-restart
+SONiC introduced a feature about auto-restarting docker containers
+if one of critical processes accidently crashed or exited. The motivation behind
+this feature is that if one critical process in a docker container crashed, then
+the whole container will not work correctly and we can try to restart to make 
+it enter into the healthy state.
+
+### Container Auto-restart show commands
+
+**show container feature autorestart**
+
+This command will display the status of auto-restart feature for all containers.
+
+- Usage:
+  ```
+  show container feature autorestart
```

Comment:
This line should show all possible ways to run the command (all optional arguments, flags, etc.). Please use the following syntax to indicate that one can specify an optional container name:

```
show container feature autorestart [<container_name>]
```

Reply:
Good suggestions! Please review it again.